performance/odm/playlist.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>odm</testCaseName>
+		<command>chmod a+x ./scripts/benchmark.sh; \
```

Comment:
are those tests have platform requirement?

Reply:
Perf test needs run on specific machine. So far I only have linux to test. The scripts and cmd need to be updated as we test on other platforms.

Reply:
Sounds good. I was thinking if <platformRequirements> is needed.

Reply:
sure, it can be add it.